.github/workflows/main.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       with:
         persist-credentials: false
     - name: Use Node from Volta
-      uses: actions/setup-node@969bd2663942d722d85b6a8626225850c2f7be4b # v3.5.0
```

Comment:
Oh, this is nice - the PR title uses the semver version (more readable) but still uses the SHA. It'd be amazing if it could change the version string, but that we may just have to live with (not bumping that as well)